### ISSUE TITLE
Databases Tables

### ISSUE THREAD

**Issue by FromBabylon** [ampache/ampache#1139]
I have an 'Error' in the Ampache test.php page for the databases Tables option. I've created the database and grant access to the user specified in the config file. 

Where can it come from ?

**Comment by Afterster** [comment]
This is definitely a database connection issue :). You probably skipped table creation at installation process.
You should start again from /install.php after removing /config/ampache.cfg.php

**Comment by FromBabylon** [comment]
I had well configured my databases manually. 
The problem was that loading localhost/ampache only loaded the requirements page. I manually accessed the /install page and everything worked (with my database already created :P) 
Didn't really understand the problem

**Comment by Afterster** [comment]
Not sure to understand neither. Looks like the first install process was partially completed. Everything ok now on your side? If so we will wait for someone else reporting the issue with additional information.

**Comment by FromBabylon** [comment]
Reading the install process, I manually enter the right values in the config file, filling the databases cred I had created in mysql. When I loaded the localhost/ampache page it only loaded the requirements pages with the Error for the Database Tables. I then try to access the install page manually but got the same thing as #100. 
So i deleted the config file, manually enter /install in the url bar and I was able to complete the install process ;-)
